pokebuild/src/components/PokemonCard.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const PokemonCard = props => {
+    return (
```

Comment:
maybe make then displayed as buttons and then add an onClick that will enlarge it to be z-index and float at the center of the page with a bigger size so you can see it